.github/ISSUE_TEMPLATE/model_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Model request
+description: Request a new model for the library
+title: "MR: "
+labels: [needs-triage, mr]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please check if your Model request is [already available](https://ollama.com/search) or that you cannot [import it](https://github.com/ollama/ollama/blob/main/docs/import.md#import-a-model) yourself.
+        Tell us about which Model you'd like to see in the library!
```

Comment:
Model probably don't need to be capitalized

Reply:
Excellent catch.